connectors/src/admin/cli.ts

```diff
@@ -45,6 +46,31 @@ import logger from "@connectors/logger/logger";
 
 const { NANGO_SLACK_CONNECTOR_ID } = process.env;
 
+async function getConnectorOrThrow({
```

Comment:
I'll refactor all the occurrences to use this helper and removes a few lines from this file.

Reply:
:+1: